csharp.NUnit/GildedRose/Program.cs

```diff
@@ -34,19 +34,15 @@ public static void Main(string[] args)
                 SellIn = 5,
                 Quality = 49
             },
+
             // this conjured item does not work properly yet
             new Item {Name = "Conjured Mana Cake", SellIn = 3, Quality = 6}
         };
 
         var app = new GildedRose(items);
 
-        int days = 2;
-        if (args.Length > 0)
-        {
-            days = int.Parse(args[0]) + 1;
-        }
 
-        for (var i = 0; i < days; i++)
+        for (var i = 0; i < 31; i++)
```

Comment:
We need dynamic days as all other texttest fixture files have them. Please keep that